src/components/PostInfo/style.css

```diff
@@ -0,0 +1,10 @@
+.header {
```

Comment:
Super, że potrafiłeś to zrobić przez imortowane css'y. Ale nie chciałbyś może skorzystać się z Material UI. Będzie to bardziej konsystentne z pozostałym kodem. Material UI ma do tego przeznaczone komponenty, na przykład: `Box` lub `Container` oraz property `sx`, w którym możesz przekazać te same style co w css klasach.